src/common/utils/constants.ts

```diff
@@ -10,3 +10,6 @@ export const FORBIDDEN_ERROR_MESSAGE
 export const INTERNAL_SERVER_ERROR_MESSAGE
   = "Something went wrong, please try again later";
 export const VALIDATION_ERROR_MESSAGE = "The validation error(s)";
+export const SERVER_ERROR = "Server side error(s)";
+export const GROUP_DOESNT_EXIST = "Group with id does not exist";
```

Comment:
How about we have a function for these messages and where and when needed we will pass the params and put them in place?

export const RESOURCE_NOT_FOUND = (resourceName: string, resourceId: string) =>  `${resourceName} with id ${resourceId} does not exist`;

And for the common messages we will have no arguments to the function just to stay consistent.

@GenieWizards/work-o-holics Any thoughts on this?

Reply:
Yes this is a good idea. This way error will be more precise and easy to build.

Reply:
@ubprudhvi Please follow this and update the PR.

Reply:
ok